src/app/views/query-response/graph-toolkit/GraphToolkit.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+
+import { Label } from 'office-ui-fabric-react';
+import { IQuery } from '../../../../types/query-runner';
+import { parseSampleUrl } from '../../../utils/sample-url-generation';
+
+const templateMap: any = {
```

Comment:
Which team will be responsible for updating/maintaining the `templateMap` ? us or the **GraphToolkit** team? I hope the  **GraphToolkit** team will do it. I propose   that we maintain the `templateMap` as a **json** file, that way, the **GraphToolkit** team can easily update the templates.

Reply:
The maintainers will be the graph toolkit team. It makes sense to move the structure to a json file